eos/b-decays/b-to-psd-l-nu.cc

```diff
@@ -426,10 +431,11 @@ namespace eos
     {
         Model::option_specification(),
         FormFactorFactory<PToP>::option_specification(),
-        { "l", { "e", "mu", "tau" }, "mu" },
-        { "U", { "c", "u" }, "c" },
-        { "q", { "u", "d", "s" }, "d" },
-    	{ "I", { "1", "0", "1/2" }, "1" },
+        { "cp-conjugate", { "true", "false" },  "false" },
+        { "l",            { "e", "mu", "tau" }, "mu"    },
+        //{ "U",          { "c", "u" },         "c"     },
```

Comment:
why keep those comments? The information is also earlier in this file in line 163.

Reply:
I would like to keep it, since we need to move it back here once ``OptionSpecification`` allows to hide an option to the user. Right now, it is hidden manually.